public/pages/MonitorDetails/containers/MonitorDetails.js

```diff
@@ -299,7 +476,28 @@ export default class MonitorDetails extends Component {
               </EuiFlexItem>
             ) : null}
           </EuiFlexItem>
-
+          <EuiFlexItem grow={false}>
+            <EuiToolTip
+              position="top"
+              content={
+                monitor.ui_metadata.search.searchType === 'graph'
+                  ? ''
+                  : 'Anomaly detector can only be automatically created from monitor defintion type "visual editor"'
+              }
+            >
+              <EuiButton
+                isLoading={updating}
+                onClick={() => this.convertToADConfigs(this.state.monitor)}
+                disabled={
+                  monitor.ui_metadata.search.searchType !== 'graph' ||
+                  monitor.ui_metadata.search.fieldName === '' ||
+                  monitor.ui_metadata.schedule.timezone
+                }
+              >
+                Create Detector
+              </EuiButton>
```

Comment:
What will the button show if a detector is already created for this monitor? Wondering if it should be changed to a link to the corresponding detector, similar to how AD handles created monitors.

Reply:
At the moment sense the monitor itself isn't directly linked with the newly made detector so the button will still show and users will have the ability to create multiple detectors from the monitor, will discuss this more with UX

Reply:
Got it, yeah this may require some backend changes to store some information on a detector created from a monitor.